### ISSUE TITLE
[Question] Can I show a graph with data from today (00:00 - now) only?

### ISSUE THREAD

**Issue by Olliek95** [kalkih/mini-graph-card#486]
Is it possible to show only data from today, starting at 00:00?
I've been looking all over for this feature, but I am unable to find anything related.

There are other questions regarding this, but they only mention 1 bar per day in a bar chart.
I'd like to show a regular graph, but only with data from today, starting at 00:00 (so not the last 24 hours).

Currently I'm seeing this (with the option; hours_to_show set to 24 hours):
![image](https://user-images.githubusercontent.com/5971963/98123534-45439100-1eb2-11eb-98b3-2db840c513b1.png)

I'd however like to only show the marked data:
![image](https://user-images.githubusercontent.com/5971963/98123619-62785f80-1eb2-11eb-8687-f7c5df743905.png)

**Comment by thejonesyboy** [comment]
Can we set `hours_to_show = 24` and then trick the renderer into thinking the time is always currently midnight tonight?

My graph currently looks like this at 9:45am:
![image](https://user-images.githubusercontent.com/885571/105434587-cda62680-5c96-11eb-8566-8b9fcdeb6959.png)

But hoping to acheive something like this (same data as above from inverter dashboard):
![image](https://user-images.githubusercontent.com/885571/105434743-1c53c080-5c97-11eb-8875-ddb8e66a0b73.png)

**Comment by thejonesyboy** [comment]
code FYI:
```
type: 'custom:mini-graph-card'
name: Solar Power (9 hours)
icon: 'mdi:weather-sunny'
points_per_hour: 10
line_width: 3
hours_to_show: 9
entities:
  - entity: sensor.power_pv_in
    name: Solar
    color: green
  - entity: sensor.power_consumption
    name: Usage
    color: red
  - entity: sensor.power_pv_in
    name: Solar
    color: green
    show_points: false
    show_legend: false
  - entity: binary_sensor.night
    color: gray
    name: Night
    show_line: false
    show_points: false
    show_legend: false
    y_axis: secondary
```

**Comment by kejinlu** [comment]
Need this feature too！

**Comment by Mediacj** [comment]
Thanks for this perfect lovelace graph! Yes it would be great to have a today possibility.

**Comment by ibennani** [comment]
This feature would be very appreciated.

**Comment by RomRider** [comment]
The other card I built in parallel has this feature:
https://github.com/RomRider/apexcharts-card 

I think this one will stay `mini` as it's name stands. 😊
I'll maintain it to keep it functional with @kalkih or add small features, but that's about it. 😊

**Comment by Olliek95** [comment]
Hi @RomRider , thanks for the info.
I've taken a look at your apexchart-card but I do feel like the look and feel of those charts is immensely different compared to the mini-graph-chart.

Since I'm a programmer by profession I've tried to tackle this myself by changing the sources.
I'll see if I can clean it up a bit and upload it as a pull request so you can take a look at it. :)
I feel like having this option is very useful while still keeping the graph "mini" like you mentioned.

**Comment by RomRider** [comment]
Pull requests are always welcome :)
For apexcharts-card, you can use the minimal layout to get somewhat the same look as mini-graph-card

**Comment by ildar170975** [comment]
Same as https://github.com/kalkih/mini-graph-card/issues/726

**Comment by Kilowatt-W** [comment]
Still a nice Feature :-)

**Comment by Disincentivato** [comment]
Hey, any news on this feature? Will be very nice to have :)

**Comment by ildar170975** [comment]
Still not a native option.
Use alternatives like config-template-card.
https://github.com/kalkih/mini-graph-card/issues/726#issuecomment-1035514883
